tests/playwright/src/model/pages/deploy-to-kubernetes-page.ts

```diff
@@ -0,0 +1,35 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import type { Locator, Page } from '@playwright/test';
+
+import { BasePage } from './base-page';
+
+export class DeployToKubernetesPage extends BasePage {
+  readonly content: Locator;
+  readonly podName: Locator;
+  readonly kubernetesContext: Locator;
+  readonly deployButton: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.content = this.page.getByRole('region', { name: 'Tab Content' });
+    this.podName = this.content.getByRole('textbox', { name: 'Pod Name' });
+    this.kubernetesContext = this.content.getByRole('textbox', { name: 'Kubernetes Context' });
+    this.deployButton = this.content.getByRole('button', { name: 'Deploy' });
+  }
+}
```

Comment:
Maybe I don't understand how it's used, but to me this looks like it's not used anywhere, why are we creating it? If the class is not used it shouldn't exist.

Reply:
You’re right! I’m working on [Issue #8053](https://github.com/containers/podman-desktop/issues/8053), which involves deploying a pod/container to a Kind cluster. I will need this class soon, so I was considering including it in the current refactoring of this page, along with locators that use the recent changes. However, it might be better to include these changes in a PR specifically for the deploying to Kind cluster scenario.